packages/server/express/src/server.ts

```diff
@@ -61,7 +61,7 @@ const defineAPI = async function (config: IConfig, storage: Storage): Promise<Ex
 
   // Hook for tests only
   if (config._debug) {
-    hookDebug(app, config.configPath);
+    hookDebug(app, config);
```

Comment:
we should display warning that the _debug mode is enabled, since we display all configuration and this file might contain sensitive data.

Reply:
i will add a check for NODE_ENV <> production and a log warning